mix.exs

```diff
@@ -1,7 +1,7 @@
 defmodule OpenIDConnect.Mixfile do
   use Mix.Project
 
-  @version "1.0.0"
+  @version "1.0.1"
```

Comment:
I don't think we publish this library, so it may be easier to leave this bump out to avoid further drift from upstream.

```suggestion
  @version "1.0.0"
```

Reply:
👍 That's fine.  I am hopeful we can move to upstream soon and not have to maintain this library at all.